tests/__init__.py

```diff
@@ -20,6 +20,8 @@
 # Make sure rate limit is enabled
 os.environ["REDASH_RATELIMIT_ENABLED"] = "true"
 
+os.environ["REDASH_ENFORCE_CSRF"] = "false"
```

Comment:
Don't we want to have some tests with CSRF? Or it's covered by Cypress?

Reply:
It felt like a behaviour that should be evaluated in an E2E perspective, so Cypress does the trick.